src/app/Recordings/ArchiveUploadModal.tsx

```diff
@@ -99,6 +118,13 @@ export const ArchiveUploadModal: React.FunctionComponent<ArchiveUploadModalProps
       title="Re-Upload Archived Recording"
       description="Select a JDK Flight Recorder file to re-upload. Files must be .jfr binary format and follow the naming convention used by Cryostat when archiving recordings."
       >
+      <CancelUploadModal
+        visible={showCancelPrompt}
+        title="Upload in Progress"
+        message="Are you sure you wish to cancel the file upload?"
+        onYes={handleAbort}
+        onNo={() => setShowCancelPrompt(false)}
+      />
```

Comment:
Nothing. It will continue to upload until explicitly cancelled.

Reply:
What if a user intended to click Yes, except the file finished uploading before they clicked it?

Reply:
Then the cancellation modal and the upload modal will disappear and the uploaded file will be displayed in the recordings list.

Reply:
Since a user would have clicked 'Cancel' or attempted to close the window before seeing the `CancelUploadModal`, what do you think about pausing the upload before a user makes the Yes/No decision?

Reply:
I don't know of a way to do that - the JavaScript "fetch" API provides a way to signal to abort the request, but not to pause a partially completed one. I'm also not sure if it would be handled well by the backend server receiving the upload - if the uploading stream of bytes suddenly stops for some time, the server may consider the request to have been disconnected or otherwise no longer valid and close the connection.